cmd/server/assets/codestatus/index.html

```diff
@@ -21,16 +21,17 @@ <h1>Verification code status</h1>
       Use an identifier to check the status of a code previously shared with your patient.
     </p>
 
-    <div class="card mb-3">
+    <div class="card mb-3" style="max-width: 30rem">
       <div class="card-header">Code status</div>
       <div class="card-body">
         <form method="POST" action="show" class="floating-form">
           {{ .csrfField }}
           <div class="form-group">
             <div class="form-label-group ">
-              <input type="text" id="uuid" name="uuid" class='form-control text-monospace{{if $code.ErrorsFor "uuid"}} is-invalid{{end}}' value="{{$code.UUID}}"
-                placeholder="UUID" maxlength="36"autocomplete="off">
-                <label for="uuid">UUID</label>
+              <input type="text" id="uuid" name="uuid"
+                class='form-control text-monospace{{if $code.ErrorsFor "uuid"}} is-invalid{{end}}'
+                value="{{$code.UUID}}" placeholder="UUID" maxlength="36" autocomplete="off">
```

Comment:
Can we drop the max-length while we're in here. I regularly lose hours of my life to fields that silently cut off my extra characters...

Reply:
wait but literally a UUID should not be longer. It's weird if you keep typing

Reply:
to elaborate: this field has as-you-type javascript that molds your input to UUID valid characters, so the length makes sense (you'll lose non-guid input already)

Reply:
okay you're right. The javascript already does all we need. This could lose data, eg. pasting with leading whitespace